### ISSUE TITLE
enable the people that get tickets assigned to set labels

### ISSUE THREAD

**Issue by lsmith77** [codetriage/codetriage#345]
github permissions are quite all or nothing but it would be cool to allow people that get tickets assigned to be able to set specific labels to help move the process forward (ie. "cannot reproduce", "has test case in a PR X" etc).

in fact it would be kind of cool to have a tool that expands the general github permission and notification system to give people certain rights over tickets based on some criteria but this might be beyond the scope of this system.

**Comment by nysthee** [comment]
I agree on the fact that this beyond the scope of this project. Although there could be a general consensus on some statuses/labels most (bigger) projects have their own system build up throughout the  years.

**Comment by schneems** [comment]
We could use the API to see what labels exist on the project and allow people to pick from that. It would be different on each project but still useful. For instance, I might want to get all issues in rails, except those that are tagged as "activerecord". Alternatively I might only want to get "actionpack" tags. I'm :+1: on a general purpose tag subscription model, we've not had anyone build it though.

**Comment by lsmith77** [comment]
yeah .. that is what I sort of had in mind. for Symfony we intend to have a label based workflow to allow users to focus on specific components. also we intend to use labels for workflow to enable people to for example label tickets as "verified" or "reviewed" to help developers with push access focus their time more on tickets that have been verified, that match coding standards etc.
